model/src/test/kotlin/JsonSchemaTest.kt

```diff
@@ -79,9 +81,9 @@ class JsonSchemaTest : StringSpec({
         errors should beEmpty()
     }
 
-    "reference.yml validates successfully" {
+    "$REFERENCE_CONFIG_FILENAME validates successfully" {
```

Comment:
Whether to use these constants in test names is probably quite a bit subjective.
I'd tend to not do that, because we want consistency which in turn means that also constants with longer names need to be used. So, the amount of characters available for providing the meat of the test case name gets reduced, which makes it harder to describe the test properly.

My proposal would be to even abstract from the actual file name in the test case name, like:
"the reference config validates successfully"

As this is opinionated I'd like to see what others think, e.g.  @oss-review-toolkit/core-devs ?

Reply:
> My proposal would be to even abstract from the actual file name in the test case name, like:
> "the reference config validates successfully"

I like this, will do.